bin/proving-service/.env

```diff
@@ -1,14 +1,12 @@
-MPS_HOST="0.0.0.0"
 MPS_PORT="8082"
-MPS_WORKERS_UPDATE_PORT="8083"
 MPS_TIMEOUT_SECS="100"
 MPS_CONNECTION_TIMEOUT_SECS="10"
 MPS_MAX_QUEUE_ITEMS="10"
 MPS_MAX_RETRIES_PER_REQUEST="1"
 MPS_MAX_REQ_PER_SEC="5"
 MPS_AVAILABLE_WORKERS_POLLING_INTERVAL_MS="20"
 MPS_HEALTH_CHECK_INTERVAL_SECS="1"
-MPS_PROMETHEUS_HOST="127.0.0.1"
 MPS_PROMETHEUS_PORT="6192"
 MPS_ENABLE_METRICS="false"
+MPS_CONTROL_PORT="8083"
```

Comment:
nit: I would move `MPS_CONTROL_PORT` up to come after `MPS_PORT`. Also, I would move `MPS_PROMETHEUS_PORT` up to come after `MPS_CONTROL_PORT`.

Also, maybe it makes sense to rename `MPS_PROMETHEUS_PORT` to `MPS_METRICS_PORT`?

Reply:
Makes sense, since it is already in the README that the format of the metrics is prometheus-compatible 👌🏼 .